olive/workflows/run/run.py

```diff
@@ -147,20 +165,28 @@ def run_engine(package_config: OlivePackageConfig, run_config: RunConfig, data_r
 
         AzureMLSystem.olive_config = run_config.to_json()
 
+    auto_optimizer_enabled = (
+        not run_config.passes
+        and run_config.auto_optimizer_config is not None
+        and not run_config.auto_optimizer_config.disable_auto_optimizer
+    )
+    if auto_optimizer_enabled:
+        is_ep_required = True
```

Comment:
This is ok for me now. But in future, if user want to specify the output model to non-onnx model, we can update this part of logics.